### ISSUE TITLE
Datapack: if user clicks hundred of times on "refresh servers"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#273]
```
Everything is downloaded and unzipped for each click...
We should avoid this.
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 8 Jan 2013 at 10:28

**Comment by GoogleCodeExporter** [comment]
```
1. first download a simple text file with a version string or maybe SHA hash. 
compare it with the last downloaded. if it's the same, don't download any data.
2. make the server compile a new global hash for each new datapack you add.

?
```

Original comment by `christian.a.reiter@gmail.com` on 9 Jan 2013 at 6:30

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision dca5baeb8f85.
```

Original comment by `eric.mae...@gmail.com` on 14 Jan 2013 at 3:36
- Changed state: **Fixed**
